app/services/manage_courses_api.rb

```diff
@@ -0,0 +1,36 @@
+module ManageCoursesAPI
+  class Connection
+    BASE_URL = Settings.manage_api.base_url
+    TOKEN = Settings.manage_api.secret
+
+    def self.api
+      Faraday.new(url: BASE_URL) do |faraday|
+        faraday.response :logger unless Rails.env.test?
+        faraday.adapter Faraday.default_adapter
+        faraday.headers['Content-Type'] = 'application/json'
+        faraday.headers['Authorization'] = "Bearer #{TOKEN}"
```

Comment:
Maybe something like this

https://github.com/DFE-Digital/manage-courses-frontend/blob/91efb2d6d592e34a9ea78625f44c790a7e528a17/app/controllers/application_controller.rb#L70

Reply:
Perfect! 👍

Reply:
Nevermind, I don't think I can add it here like that.